### PR TITLE
fix deserialize power field in validator_update

### PR DESCRIPTION
**Pull request by yihuang**
When removing validator, the power field is set to zero and ignored in json format.

### CODE REVIEW

File: tendermint/src/validator.rs [hunk 1]
```diff
@@ -192,6 +192,7 @@ pub struct Update {
     pub pub_key: PublicKey,
 
     /// New voting power
+    #[serde(default)]
```
Comment (liamsi) [inline tendermint/src/validator.rs:195]: Ouch, thanks! I'm wondering if there shouldn't be two test cases (one with power and one without) instead of one. This is all so brittle :-/ We do not test all edge cases well. ref #303
